docs/modules/ROOT/pages/how-tos/deleting.adoc

```diff
@@ -1 +1,41 @@
-= Deleting applications and components
\ No newline at end of file
+= Deleting applications and components
+
+You can delete an application if you are the owner or if you have admin access to the application.
```

Comment:
I would not say owner or admin access. Instead, you can say something like if you are a user with the workspace admin role.